src/Microsoft.Identity.Web/EasyAuth/AppServiceAuthenticationInformation.cs

```diff
@@ -0,0 +1,96 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.Identity.Web
+{
+    /// <summary>
+    /// Information about the AppService configuration on the host.
+    /// </summary>
+    public static class AppServiceAuthenticationInformation
+    {
+        // Environment variables.
+        private const string EasyAuthEnabledEnvironmentVariable = "WEBSITE_AUTH_ENABLED";            // True
+        private const string EasyAuthOpenIdIssuerEnvironmentVariable = "WEBSITE_AUTH_OPENID_ISSUER"; // for instance https://sts.windows.net/<tenantId>/
+        private const string EasyAuthClientIdEnvironmentVariable = "WEBSITE_AUTH_CLIENT_ID";         // A GUID
+        private const string EasyAuthClientSecretEnvironmentVariable = "WEBSITE_AUTH_CLIENT_SECRET"; // A string
+        private const string EasyAuthLogoutPathEnvironmentVariable = "WEBSITE_AUTH_LOGOUT_PATH";    // /.auth/logout
+        private const string EasyAuthIdentityProviderEnvironmentVariable = "WEBSITE_AUTH_DEFAULT_PROVIDER"; // AzureActiveDirectory
+        private const string EasyAuthAzureActiveDirectory = "AzureActiveDirectory";
+
+        // Artificially added by Microsoft.Identity.Web to help debugging Easy Auth. See the Debug controller of the test app
+        private const string EasyAuthDebugHeadersEnvironmentVariable = "EASY_AUTH_LOCAL_DEBUG";
+
+        /// <summary>
+        /// Is AppService authentication enabled?.
+        /// </summary>
+        public static bool IsAppServiceAadAuthenticationEnabled
+        {
+            get
+            {
+                return (Environment.GetEnvironmentVariable(EasyAuthEnabledEnvironmentVariable) == "True")
+                    && Environment.GetEnvironmentVariable(EasyAuthIdentityProviderEnvironmentVariable) == EasyAuthAzureActiveDirectory;
+            }
+        }
```

Comment:
as we discussed yesterday, will there be an error w/aka.ms link for customers using FB or another IDP that's not AAD?

Reply:
I added an issue to do it later: https://github.com/AzureAD/microsoft-identity-web/issues/702